.gitlab/kernel_matrix_testing/system_probe.yml

```diff
@@ -270,6 +273,8 @@ kernel_matrix_testing_run_system_probe_tests_arm64:
           - "centos_79"
           - "centos_8"
         TEST_SET: ["no_tracersuite", "only_tracersuite"]
+  after_script:
+    - !reference [.collect_junit_kmt]
```

Comment:
I think this job can use the `x64` buildimage. Just leave the `ARCH` variable in place and maybe create a `CONNECTOR_ARCH` variable so the correct `connector` variant gets used that matches the buildimage arch. That way you can get rid of the specific `arm64` upload job.

Reply:
I'm not sure the `CONNECTOR_ARCH` is needed, so I've done a test without it, and I'll check if it works. Thanks for the advice is would simplify the config

Reply:
It should be needed because the runner will be `x64`, but the metal instance will still be `arm64`.

Reply:
This is the line that would cause problems: `$CI_PROJECT_DIR/connector-$ARCH -host $INSTANCE_IP -user ubuntu -ssh-file $AWS_EC2_SSH_KEY_FILE -vm-cmd "${NESTED_VM_CMD}"`. It runs on the runner, so if the `ARCH` doesn't match the runner architecture, then it wouldn't work.

Reply:
Are you ok if I do this as a PR follow-up?

Reply:
Sure

Reply:
understood, I'll do another try, thanks